lib/LaTeXML/MathParser.pm

```diff
@@ -561,9 +563,9 @@ sub parse_kludge {
   my $wrapped_replacements;
```

Comment:
why not just add `_unparsed` to all the `@replacements` and not bother wrapping them at all?

Reply:
Oh, sure.

Reply:
Done.